apps/mdn/mdn-aws/infra_testing/tests/test_listeners.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+
+import boto3
+from kubernetes import client, config
+import os
+import pytest
+
+# fixtures starting with k8s_ return K8s API objects
+#   documented here: https://github.com/kubernetes-incubator/client-python/blob/master/kubernetes/README.md
+# fixtures starting with aws_ return boto3 objects
+#   documented here: http://boto3.readthedocs.io/en/latest/index.html
+
+
+@pytest.fixture(scope="module")
+def k8s():
+    """A Kubernetes client"""
+    config.load_kube_config(os.environ['KUBECONFIG'])
```

Comment:
I don't normally set a KUBECONFIG env var and use the default file location instead. How about we use `os.environ.get('KUBECONFIG')` here instead?

Reply:
👍